rllib/algorithms/algorithm_config.py

```diff
@@ -425,6 +419,8 @@ def __init__(self, algo_class=None):
         # `self.rl_module()`
         self.rl_module_spec = None
         self._enable_rl_module_api = False
+        # Whether to error out if exploration config is set when using RLModules.
+        self._validate_exploration_conf_and_rl_modules = True
```

Comment:
This option makes it possible to disable the Error on "_enable_rl_module_api" and "exploration_config!={}".
This is useful because we need to circumvent this temporarily when switching to `_enable_rl_module_api=True` in the PPOConfig, but still doing `_enable_rl_module_api=False` in some other places where these test would be red otherwise.